features/htcondor/params.pan

```diff
@@ -60,6 +60,16 @@ variable CONDOR_CONFIG = {
     SELF['config.local'] = '';
   };
 
+  # Default configuration is *not* enable Multicore
+  if (!is_defined(SELF['multicore'])){
```

Comment:
are these defaults? it's better to make a structire template, and then fill in like
```pan
structure template whatever/htcondordefault;
"a" = "b"; 
```

and then here
```
foreach(k;v;create('whatever/htcondordefault')) {
    if (!is_defined(SELF[k])) { SELF[k]=v;};
}
```